LocationReminders/LocationReminders/LocationController.m

```diff
@@ -0,0 +1,59 @@
+//
+//  LocationController.m
+//  LocationReminders
+//
+//  Created by Christina Lee on 5/2/17.
+//  Copyright © 2017 Christina Lee. All rights reserved.
+//
+
+#import "LocationController.h"
+@import MapKit;
+@import CoreLocation;
+
+@interface LocationController() <CLLocationManagerDelegate, LocationControllerDelegate>
```

Comment:
This file doesn't need to conform to the `LocationControllerDelegate`, just the `CLLocationManagerDelegate`.